nuxeo-connect-client/src/main/java/org/nuxeo/connect/packages/dependencies/DependencyResolver.java

```diff
@@ -57,4 +57,29 @@ public DependencyResolution resolve(List<String> pkgInstall,
             String targetPlatform, boolean allowSNAPSHOT, boolean doKeep)
                     throws DependencyException;
 
+    /**
+     * Compute a {@link DependencyResolution} that will match the requested packages installation, remove and upgrade
```

Comment:
(detail, for future commits) The first sentence of a Javadoc is used for the API summary and should be shorter, written in the indicative (not imperative), and ended with a period. `Resolves dependencies for the given packages. Computes a {@link DependencyResolution} ...`